qubes-rpc/admin.vm.Console

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+lock="/var/run/qubes/$QREXEC_REQUESTED_TARGET.terminal.lock"
+
```

Comment:
This service (especially when called from mgmt vm) should (also?) check if domain is running. Maybe the check should be moved here instead?

Another thing is, all `admin.*` calls normally go to qubesd and (among other things) emit `mgmt-permission:...` event, to allow addons to plug some more permission checks. I don't think it's a blocker to have it here, but please add a TODO comment here. And maybe create an issue on github, to not forget about it.